src/modules/client/providers/ClientSsnDobVisibility.tsx

```diff
@@ -71,12 +70,9 @@ export const ContextualSsnToggleButton: React.FC<ButtonProps> = (props) => (
           {...props}
           on={show}
           onToggle={() => setShow((prev) => (isNil(prev) ? true : !prev))}
-          aria-hidden
         >
           <strong>SSN</strong>
         </VisibilityToggleButton>
```

Comment:
This reverts a change introduced in an a11y fix released last year: https://github.com/greenriver/hmis-frontend/pull/941#discussion_r1771936182

We waffled about it at the time, so given that it's causing automated tests to now fail, I would be in favor of reverting it. We should have tried running the axe tools in browser at the time 🤦‍♀️  (and hopefully automated tests will help us catch this kind of thing in the future!)